ApollosProject/apollos-apps
# chore: Community Tab Empty State

## ✏️ Description
If a user is not a part of any communities, we want to show them a unique screen that invites them to participate in a Community.

In order to do this, we check to see if the user is in any communities and if they aren't we render a unique component instead of the Stream Activity Feed. When a user joins/leaves a Community, this screen adjusts appropriate to the number of Communities the user is in.

## 🔬 To Test
1. Open up the Communities Tab
2. Remove yourself from any communities you are a member of
3. Check to see the Communities tab now shows the custom empty state
4. Join a community
5. Check to see that the communities tab now updates to show communtiies

## 📸 Screenshots

https://user-images.githubusercontent.com/45076058/198084215-302bb907-128b-4be4-a24b-f7151ff20314.mp4


## ⚠️ To-do Before Merge
- [x] Ensure that #2734 and #2721 are merged into `master`

## 📝 Checklist
